pkg/client/package.json

```diff
@@ -38,6 +39,7 @@
     "keycloak-js": "12.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-hook-form": "^7.27.1",
```

Comment:
Why add this when we have formik?

Reply:
React-hook-form is much more efficient both in term of library size and rendering times.
So ultimately all formik code to be refactored using it.